plugin/rubocop.vim

```diff
@@ -54,8 +54,12 @@ function! s:RuboCop(current_args)
   endif
   let l:rubocop_output  = substitute(l:rubocop_output, '\\"', "'", 'g')
   let l:rubocop_results = split(l:rubocop_output, "\n")
-  cexpr l:rubocop_results
-  copen
+  if len(l:rubocop_results)
```

Comment:
I think `cwindow` in essence does this same logic; if there are errors populated in `cexpr`, then it opens the `cwindow` if it is not opened. Else, it does nothing

Reply:
Heh, I forgot all about this PR and have been just maintaining my own fork.

`cwindow` doesn't work here since it's just parsing the return value of a `system()` call which does not populate the quickfix list.